.github/workflows/ci_cd.yml

```
@@ -0,0 +1,53 @@
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v4
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'npm'
+
+    - name: Install pnpm
+      run: pnpm/action-setup@v4    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)